tests/testsuite.src/syn_copy.at

```diff
@@ -577,3 +651,33 @@ AT_CHECK([$COMPILE -free prog.cob], [0], [], [])
 AT_CHECK([$COBCRUN_DIRECT ./prog], [0], [OK], [])
 
 AT_CLEANUP
+
+
+AT_SETUP([REPLACE: partial replacement with SPACES figurative constant])
+AT_KEYWORDS([replace gcos])
```

Comment:
add "acu" too, please :-)

Reply:
That's good, I've meant "because it is in acu conf, use `AT_KEYWORDS([replace gcos acu])`" :-)